setup.py

```diff
@@ -13,63 +12,44 @@
           '-Wno-cast-function-type', '-std=c99', '-D_FILE_OFFSET_BITS=64']
 
 alpm = Extension('pyalpm',
-    libraries=['alpm'],
-    extra_compile_args=cflags + ['-DVERSION="%s"' % pyalpm_version],
-    language='C',
-    sources=[
-        'src/pyalpm.c',
-        'src/util.c',
-        'src/package.c',
-        'src/db.c',
-        'src/options.c',
-        'src/handle.c',
-        'src/transaction.c'
-        ],
-    depends=[
-        'src/handle.h',
-        'src/db.h',
-        'src/options.h',
-        'src/package.h',
-        'src/pyalpm.h',
-        'src/util.h',
-        ])
-
-class TestCommand(Command):
-    user_options = []
-
-    def initialize_options(self):
-        pass
-
-    def finalize_options(self):
-        pass
-
-    def run(self):
-        raise SystemExit(
-            subprocess.call(['nosetests',
-                             'tests']))
+                 libraries=['alpm'],
+                 extra_compile_args=cflags + ['-DVERSION="%s"' % pyalpm_version],
+                 language='C',
+                 sources=['src/pyalpm.c',
+                          'src/util.c',
+                          'src/package.c',
+                          'src/db.c',
+                          'src/options.c',
+                          'src/handle.c',
+                          'src/transaction.c'],
+                 depends=['src/handle.h',
+                          'src/db.h',
+                          'src/options.h',
+                          'src/package.h',
+                          'src/pyalpm.h',
+                          'src/util.h'])
 
 
 with open("README", "r") as fh:
     long_description = fh.read()
 
+pycman_scripts = ['database', 'deptest', 'query', 'remove', 'sync', 'upgrade', 'version']
+
 setup(name='pyalpm',
       version=pyalpm_version,
       description='libalpm bindings for Python 3',
       long_description=long_description,
+      long_description_content_type="text/markdown",
       author="Rémy Oudompheng",
       author_email="remy@archlinux.org",
       url="https://projects.archlinux.org/pyalpm.git",
       packages=["pycman"],
-      scripts=["scripts/lsoptdepends"] + ["scripts/pycman-" + i
-          for i in ['database', 'deptest', 'query', 'remove', 'sync', 'upgrade', 'version']],
+      scripts=["scripts/lsoptdepends"] + [f'scripts/pycman-{p}' for p in pycman_scripts],
       ext_modules=[alpm],
-      cmdclass={
-          'test': TestCommand
-      },
       classifiers=[
-        'Development Status :: 6 - Mature',
-        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
-        'Programming Language :: C',
+          'Development Status :: 6 - Mature',
+          'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+          'Programming Language :: C',
```

Comment:
I also wonder if we want to drop:
```
'Topic :: System :: Software Distribution '
'Topic :: System :: Systems Administration'
```

In there.

Reply:
sounds good